__init__.py

```diff
@@ -22,10 +22,10 @@ def __init__(self):
 
     def _connect(self, message):
         url = 'http://localhost:6680'
-        if self.settings:
-            url = self.settings.get('mopidy_url', url)
-        if self.config:
-            url = self.config.get('mopidy_url', url)
+#        if self.settings:
+#            url = self.settings.get('mopidy_url', url)
+#        if self.config:
+#            url = self.config.get('mopidy_url', url)
```

Comment:
Did you intend to comment out this block to get mopidy url?

Reply:
Ah, I had forgot to revisit that bug.  Yes I meant to comment it out initially as for me at least it couldn't get config and so the skill didn't connect to mopidy.